subprojects/packagefiles/theora/lib/meson.build

```diff
@@ -0,0 +1,204 @@
+encoder_sources = files(
+    'apiwrapper.c',
```

Comment:
Typical meson style is to use 2 spaces for indentation. My muon fmt config does that as well.